src/pages/graph-config/saveGraphConfig.html

```diff
@@ -21,6 +21,16 @@ <h1>
         <div class="input-group input-group-lg">
             <input required class="form-control graph-config-name" type="text" placeholder="My graph config (required)" ng-model="newConfig.name" >
         </div>
+        <br/>
+        <p>Description</p>
+        <div class="input-group input-group-lg">
+            <input width="100" class="form-control graph-config-description" type="text" placeholder="Enter a description" ng-model="newConfig.description" >
+        </div>
+        <br/>
+        <p>Example</p>
```

Comment:
This should be visible only when the visual graph starting point is "Start with a search box"